app/services/data_requests/create_email_addresses_report.rb

```diff
@@ -0,0 +1,19 @@
+module DataRequests
+  class CreateEmailAddressesReport
+    attr_reader :user
+
+    def initialize(user)
+      @user = user
+    end
+
+    def call
+      user.email_addresses.map do |email_address|
```

Comment:
do we want to like, CSV format any of these reports? or are they fine as these hashes?

Reply:
The way I have been doing this is running a script on the host that dumps a bunch of data into JSON which I then move off the host and process on my machine. I keep things in JSON because it is easier to move and process structured data.